cli/__init__.py

```diff
@@ -0,0 +1,13 @@
+__app_name__ = "empiric_cli"
+__version__ = "0.1.0"
```

Comment:
let's use the same version as empiric-package please

Reply:
in that case... do we want to package this with the empiric package as a submodule?

Reply:
Yes, that would be great